src/validation/error.messages.ts

```diff
@@ -47,13 +47,16 @@ export enum ErrorMessages {
   SELECT_IF_MANAGING_OFFICER_REGISTER_IN_COUNTRY_FORMED_IN = "Select yes if the corporate managing officer is already on a public register in the country it was formed",
   SELECT_IF_SERVICE_ADDRESS_SAME_AS_PRINCIPAL_ADDRESS = "Select yes if the correspondence address the same as the principal or registered office address",
   SELECT_IF_MANAGING_OFFICER_SERVICE_ADDRESS_SAME_AS_PRINCIPAL_ADDRESS = "Select yes if the corporate managing officer’s correspondence address is the same as the principal or registered office address",
-  SELECT_IF_SERVICE_ADDRESS_SAME_AS_USER_RESIDENTIAL_ADDRESS = "Select yes if the individual person’s correspondence address is the same as their home address",
+  SELECT_IF_SERVICE_ADDRESS_SAME_AS_USER_RESIDENTIAL_ADDRESS = "Select yes if the correspondence address is the same as their home address",
   SELECT_IF_ANY_BENEFICIAL_OWNERS_BEEN_IDENTIFIED = "Select if any beneficial owners have been identified",
   SELECT_THE_TYPE_OF_BENEFICIAL_OWNER_OR_MANAGING_OFFICER_YOU_WANT_TO_ADD = "Select the type of beneficial owner or managing officer you want to add",
   SELECT_IF_SECURE_REGISTER_FILTER = "Select yes if any of the entity’s beneficial owners have ever applied to protect personal information at Companies House",
   SELECT_WHO_IS_MAKING_FILING = "Select who is completing this registration",
   CHECK_DILIGENCE = "Check and confirm the statement of compliance",
-  // MAX Lengths
+  SELECT_IF_ON_SANCTIONS_LIST = "Select yes if the individual person is on the sanctions list",
```

Comment:
Discussed this with @markpit . We agreed to go for a generic sanctions list message so that can be used on other pages (a bit like we've done with other generic fields). Could this text please change to "Select yes if on the sanctions list"? Thanks.

Reply:
yes, if it has been agreed, my branch still had the specific messages

Reply:
Updated noc code having tested the code on[ ](https://github.com/companieshouse/overseas-entities-web/compare/roe-383-bo-other-legal-entity-validation)